### ISSUE TITLE
Quick documentation reference in ExampleCompany.org

### ISSUE THREAD

**Issue by jamesread** [jamesread/orgviz#13]
Lindsay Cassidy recommended this feature, which I think is great. Simply, to have a quick documentation reference in ExampleCompany.org, making it more useful as a starting point;

````
# Tag Options:
# Full Name
#	title: Job Title
#	team: Team Name
#	country: Country Name
#	supports -> Full Name	
#	reports -> Full Name
#	dmu: <U,I,G,B,D>
#	sentiment: <P,N,O>
#	influence: <E,S,P,I>
#
# Note:
# dmu: <User, Influencer, Gatekeeper, Buyer, Decision Maker>
# sentiment: <Proponant, Neutral, Opponent>
# influence: <Enemy(red), Supporter(blue), Promoter(green), Internal(black)>
````

I would like to save this fix/commit for Lindsay himself, so he shows up on the contributors list :-)

**Comment by Fale** [comment]
The list is missing the `id` option  ;-)

**Comment by jamesread** [comment]
That is because Lindsay wrote the description, and he didn't know about the `id` option :-) I've got a meeting with Lindsay on Tuesday to help him commit this change himself :-)

**Comment by jamesread** [comment]
This is closed as of PR #16 .
